frontend/src/container/NotificationWall.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+import classnames from 'classnames'
+import { translate } from 'react-i18next'
+import { getNotificationList } from '../action-creator.async.js'
+import {
+  setNotificationList,
+  updateNotification
+} from '../action-creator.sync.js'
+import {
+  displayDistanceDate,
+  GenericButton,
+  ListItemWrapper,
+  PopinFixedHeader,
+  TracimComponent
+} from 'tracim_frontend_lib'
+
+export class NotificationWall extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      isNotificationWallOpen: true,
+      hasMoreNotifications: true
+    }
+  }
+
+  async componentDidMount () {
+    const { props } = this
+
+    const fetchGetNotificationWall = await props.dispatch(getNotificationList(props.user.userId))
+    switch (fetchGetNotificationWall.status) {
+      case 200:
+        props.dispatch(setNotificationList(fetchGetNotificationWall.json))
+        break
+      default:
+        this.sendGlobalFlashMessage(props.t('Error while loading notification list'))
+    }
+  }
+
+  handleClickBtnClose = notificationId => {
+    const { props } = this
+    const notification = props.notificationList.find(notification => notification.id === notificationId)
+    props.dispatch(updateNotification({ ...notification, read: true }))
+    this.setState({ isNotificationWallOpen: false })
+  }
+
+  handleClickSeeMore = () => { }
+
+  getNotificationTypeText = notification => {
+    const { props } = this
+    switch (notification.type) {
+      case 'content.created':
```

Comment:
I didn't think of using them precisely because they are notifications and not TLMs, but they are built almost the same way, so I agree that I should have used them, I will adapt the development for that. The only difference I think I will keep (and don't be sorry to tell me if you find a bad idea) is that since the notifications don't have specific contain related to the type of content, I'm not taking into account the optionalSubType, e.g. all the types of content created will be in the same variable so when we add new types it won't be necessary to change the notifications too.

Reply:
I'm fine with that.

Another thoughts: could it be that the real issue is the name of the constants used for TLM ? If we rename them in something more generic, could we use them for both purpose ?
The new name could be to just remove the "LIVE MESSAGE" part to match the backend definition of "messages"

Reply:
Keeping LIVE in the name doesn't bother me since the notifications come from the TLMs. I think it's okay to keep the name and still use it for both.